### DISCUSSION TITLE
Welcome to Soul-of-Waifu Discussions!

### DISCUSSION THREAD

**Discussion by jofizcd** [jofizcd/Soul-of-Waifu#3] [Announcements]
## 👋 Welcome!
  We’re using a Soul-of-Waifu Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vanuhabarskij-cyber** [comment]
Когда будет поддержка Linux?

**Reply by CrazyAngelm** [reply]
Я бы её даже мог реализовать. Но проблема в том, что автор вообще не одного pr в принципе не принимает в проект. Поэтому как говориться пусть пашет сам.
